tests/Feature/Api/V1/UserWantToPlayListTest.php

```diff
@@ -0,0 +1,315 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature\Api\V1;
+
+use App\Community\Enums\UserGameListType;
+use App\Models\Game;
+use App\Models\System;
+use App\Models\User;
+use App\Models\UserGameListEntry;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Carbon;
+use Tests\TestCase;
+
+class UserWantToPlayListTest extends TestCase
+{
+    use RefreshDatabase;
+    use BootstrapsApiV1;
+
+    public function testItValidates(): void
+    {
+        $this->get($this->apiUrl('GetUserWantToPlayList'))
+            ->assertJsonValidationErrors([
+                'u',
+            ]);
+    }
+
+    public function testGetUserWantToPlayListUnknownUser(): void
+    {
+        $this->get($this->apiUrl('GetUserWantToPlayList', ['u' => 'nonExistant']))
+            ->assertSuccessful()
+            ->assertJson([]);
+    }
+
+    public function testGetUserWantToPlayList(): void
```

Comment:
nit: Recommend cutting this from 10 games to 5. The validation will be just as effective and it'll make the test a little easier for a reader to skim.